4.classify-single-cell-phenotypes/classification-utils.py

```diff
@@ -8,70 +8,29 @@
 from sklearn.linear_model import LogisticRegression
 
 
-def classify_plate_cells(classifier: LogisticRegression, plate_load_path: pathlib.Path) -> pd.DataFrame:
-    """
-    classify cells in plate
-
-    Parameters
-    ----------
-    classifier :
-        sklearn LogisticRegression classifier
-    plate_load_path : pathlib.Path
-        path to plate to load and classify
-
-    Returns
-    -------
-    plate_classifications : pd.DataFrame
-        single cell dataframe with metadata and phenotypic class probabilities
-    """
-
-    # load in one row to create datatypes dictionary for faster loading
-    first_row_data = pd.read_csv(plate_load_path, compression="gzip", nrows=1)
-    metadata_cols = [col for col in first_row_data.columns if "efficientnet" not in col]
-    feature_cols = [col for col in first_row_data.columns if "efficientnet" in col]
-
-    # specify datatypes for metadata/feature columns
-    metadata_dtypes = {metadata_col: str for metadata_col in metadata_cols}
-    feature_dtypes = {feature_col: np.float32 for feature_col in feature_cols}
-    # combine both dictionaries
-    plate_dtypes = {**metadata_dtypes, **feature_dtypes}
-
-    # load in csv with the specified column names/datatypes
-    plate_data = pd.read_csv(
-        plate_load_path, compression="gzip", dtype=plate_dtypes, low_memory=True
-    )
-
-    # combine metadatas and classifications dataframes
-    cell_metadatas = plate_data[metadata_cols].reset_index(drop=True)
-    cell_features = plate_data[feature_cols].values
-    cell_classifications = pd.DataFrame(
-        classifier.predict_proba(cell_features),
-        columns=classifier.classes_,
-    ).reset_index(drop=True)
-    return pd.concat([cell_metadatas, cell_classifications], axis=1)
-
-
-def save_feature_classifications(phenotypic_profiling_model: LogisticRegression, normalized_plates_path: pathlib.Path, output_dir: pathlib.Path):
-    """
-    classify features from normalized plates and save these classifications
-
-    Parameters
-    ----------
-    phenotypic_profiling_model : LogisticRegression
-        model to use for classification
-    normalized_plates_path : pathlib.Path
-        path to normalized plates
-    output_dir : pathlib.Path
-        where to save features classifications
-    """
+def get_probas_dataframe(plate_features, model, feature_type):
```

Comment:
Recommend adding type hints here.

Reply:
Done in https://github.com/WayScience/cell-health-data/pull/16/commits/87bf3b49e57b36410caba8612063de42d8f265e2